src/components/common/DetailComment.vue

```diff
@@ -1,17 +1,19 @@
 <template>
   <v-card flat id="detail-comment-container">
-    <v-textarea
-      ref="textarea"
-      auto-grow
-      rows="5"
-      id="detail-comment-textarea"
-      :counter="maxLength"
-      :rules="rules"
-      :value="value"
-      :placeholder="placeholder"
-      :autofocus="autofocus"
-      @input="emitInput($event)"
-    />
+    <v-row no-gutters class="pl-4 pr-4 pt-2 pb-2">
+      <v-textarea
+        ref="textarea"
+        auto-grow
+        rows="5"
+        id="detail-comment-textarea"
+        :counter="maxLength"
+        :rules="rules"
+        :value="value"
+        :placeholder="placeholder"
+        :autofocus="autofocus"
+        @input="emitInput($event)"
+      />
+    </v-row>
```

Comment:
This DetailComment component is used by various filings. You need to check all of them to ensure this extra padding doesn't mess those up.

PS Check out the [Vuetify spacing helper classes](https://vuetifyjs.com/en/styles/spacing/) -- you could use py-something here.

Reply:
Good call, I only checked the `DetailComment` for corrections. I'll track down where else it is used and make sure the extra padding doesn't affect it. 

Also yeahI think it would look cleaner using `px-#` and `py-#` since the vertical/horizontal numbers are the same.